### PR TITLE
(DOCS-6995) Add Postman auth note

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do? What is the motivation?
Add a note about using the Datadog authentication environment with the Datadog Postman collection

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/api/latest/_index.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ To try out the API [![Run in Postman][3]](https://god.gw.postman.com/run-collect
 
 [Using the API][4] is a guide to the endpoints.
 
-**Note**: cURL code examples assume usage of BASH and GNU coreutils. On macOS, you can install coreutils via the [Homebrew package manager][5]: `brew install coreutils`
+**Notes**: 
+   - Configure the Datadog Postman collection's included Datadog Authentication environment with your API and application key.
```
Comment (estherk15) [inline content/en/api/latest/_index.md:36]: Non-blocking
```suggestion
   - Configure the Datadog Postman collection's Datadog Authentication environment with your API and application key.
```

File: content/en/api/latest/_index.md [hunk 2]
```diff
@@ -32,7 +32,9 @@ To try out the API [![Run in Postman][3]](https://god.gw.postman.com/run-collect
...
+   - cURL code examples assume usage of BASH and GNU coreutils. On macOS, you can install coreutils via the [Homebrew package manager][5]: `brew install coreutils`
```
Comment (estherk15) [inline content/en/api/latest/_index.md:37]:
```suggestion
   - cURL code examples assume usage of BASH and GNU coreutils. On macOS, you can install coreutils with the [Homebrew package manager][5]: `brew install coreutils`
```

File: content/en/getting_started/api/_index.md [hunk 3]
```diff
@@ -47,6 +47,8 @@ data-postman-param="env%5BDatadog%20Authentication%5D=W3sia2V5IjoiYXBpX2tleSIsIn
 
 </br>This collection works in Postman for Web or in your Postman application. It may take several seconds to load.
 
+**Note**: Configure the Datadog Postman collection's included Datadog Authentication environment with your API and application key.
```
Comment (estherk15) [inline content/en/getting_started/api/_index.md:50]: Non-blocking
```suggestion
**Note**: Configure the Datadog Postman collection's Datadog Authentication environment with your API and application key.
```

File: content/en/api/latest/_index.md (hunk 2 above)
Comment (buraizu) [reply content/en/api/latest/_index.md:37]: Thanks for catching this!

**Pull request merged**